settings/provisioner/openstack/neutron/default.yml

```diff
@@ -0,0 +1,38 @@
+---
+# This is the most common network topology for OpenStack Testing deployments
+
+networks:
+    data:
+        name: '{{ random.uuid }}-data'
```

Comment:
I suggest we add the input argumet "prefix" if users wish to avoid the randomizer (ie for cleanup):
Then, we can do:
`'{{ provisioner.prefix | default(random.uuid) }}-data'`

That will try to use "prefix", and if missing, use the random prefix